### ISSUE TITLE
Access to puppeteer page element

### ISSUE THREAD

**Issue by tavyandy97** [frinyvonnick/node-html-to-image#61]
Is there any way to access the page element from puppeteer in node-html-to-image. For eg. 
```
const bodyWidth = await page.evaluate(() => document.body.scrollWidth);
const bodyHeight = await page.evaluate(() => document.getElementById("wrapper").clientHeight);
await page.setViewport({ width: bodyWidth, height: bodyHeight, deviceScaleFactor: 2 });
```

If not can it be added? Maybe in the form of a function where we can access the page element, just before taking a screenshot.

**Comment by frinyvonnick** [comment]
Hi @tavyandy97 👋 

There isn't any way to access page element for the moment. The function is a good idea in my opinion. We could call this option `beforeScreenshot`. Would you be interested in implementing this feature?

**Comment by tavyandy97** [comment]
@frinyvonnick Sure I'd be happy to contribute!

**Comment by frinyvonnick** [comment]
Can't wait to see your PR 😉 If you have any questions, don't hesitate to ask 👌

**Comment by frinyvonnick** [comment]
Fixed by #65
